packages/discovery-provider/plugins/pedalboard/apps/relay/src/middleware/logging.ts

```diff
@@ -26,11 +26,12 @@ export const outgoingLog = (request: Request, response: Response) => {
     new Date().getTime() - response.locals.ctx.startTime.getTime();
   const { route, method } = request;
   const { locals: ctx } = response;
+  const requestId = ctx.ctx.requestId
   const statusCode = response.statusCode;
   const path: string = route.path
   if (!path.includes("health")) {
     logger.info(
-      { route, method, abi: ctx.ctx.validatedRelayRequest.encodedABI, responseTime, statusCode },
+      { requestId, path, method, responseTime, statusCode },
```

Comment:
no abi?

Reply:
oh good catch, thought I had that in other logs

Reply:
added it into the validation logs so the abi is recorded early and we can track it all by request id